feat: add faucet links to testnet page

ffe9f8 opened the pull request.

StephenFluin reviewed (changes requested):
Rather than 'alfajores' use the well-known name of the chain or token, such as  'Celo Alfajores' .

Did you try all of these? The Paradigm one doesn't work for me.

Why an H3 and not an H2? I think either might be okay, I just want to make sure you thought about it.

ffe9f8 commented:
Rather than 'alfajores' use the well-known name of the chain or token, such as 'Celo Alfajores' .
> Done.

Did you try all of these? The Paradigm one doesn't work for me.
> Resolved in meeting. Removed links that do not work.

Why an H3 and not an H2? I think either might be okay, I just want to make sure you thought about it.
> Good catch, was copying/pasting formatting from another section and didn't notice this. Changed to H2 and moved the section higher so that developers can see it earlier on. (I think that there could be more visual distinction between an H3 and an H2, will put it in the style guide.)
